packages/graphql/tests/integration/optional-features-int.test.ts

```diff
@@ -0,0 +1,136 @@
+/*
```

Comment:
**suggestion:** I would say this file is completely unnecessary, as this is totally covered by [packages/graphql/tests/schema/string-comparators.test.ts](https://github.com/neo4j/graphql/pull/1773/files#diff-dd5f7d162b2d2fa7ee4b30ea50e73942b783c8cba16dd3e4244143fbb9dd013f)! 🙂

Reply:
I think you're right 😄